src/index.html

```diff
@@ -1,13 +1,335 @@
+                </h3>
+                <div class="testimonial__role">Senior Vice President</div>
+                <p class="testimonial__about">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart</p>
+              </div>
+            </div>
+            <div class="testimonials__item grid__item grid__item--9-12">
+              <div class="testimonial">
+                <div class="testimonial__photo-container">
+                  <img
+                    src="./images/people/3.jpg"
+                    alt="Leslie Mckinney"
+                    class="testimonial__photo"
+                  >
+                </div>
+                <h3 class="testimonial__name">
+                  <a href="#" class="link">Max Henry</a>
+                </h3>
+                <div class="testimonial__role">Chief Design Officer</div>
+                <p class="testimonial__about">When my lovely valley of steam rises and half-day sun is above an impermeable</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">Phone</div>
+                <a href="tel: +1 234 5555-55-55" class="contact-us__info-link">+1 234 5555-55-55</a>
+              </div>
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">Email</div>
+                <a href="mailto: hello@miami.com" class="contact-us__info-link">hello@miami.com</a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">Address</div>
+                <a href="https://goo.gl/maps/4RiG6oo9XciScf2H9" class="contact-us__info-link">
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--8-12">
+              <form action="">
+                <input
+                  name="name"
+                  type="text"
+                  class="contact-us__field form-field"
+                  placeholder="Name"
+                >
+                <input
+                  name="email"
+                  type="email"
+                  class="contact-us__field form-field"
+                  placeholder="Email"
+                >
+                <textarea
+                  name="message"
```

Comment:
Let's fix this textarea (here the video https://www.loom.com/share/a46ddc66202c43838fdbbfb1851e3e71)
Because now it looks strange without placeholder and strange text position

<img width="725" alt="Снимок экрана 2021-07-21 в 16 57 44" src="https://user-images.githubusercontent.com/66570142/126798218-364a6f12-9484-41d1-ada1-3275c2fd5d52.png">

<img width="585" alt="Снимок экрана 2021-07-23 в 17 35 27" src="https://user-images.githubusercontent.com/66570142/126798297-0cef6996-8538-48fc-932c-1268d8970529.png">